### ISSUE TITLE
Is the software still maintained?

### ISSUE THREAD

**Issue by Nastras** [take4blue/PrintControler#1]
Hello @take4blue,

I came across this software on your homepage and find it really great. I have already tested version 1.4. Unfortunately, there is no link on your homepage to download the latest version 1.6.1.

Is there a possibility to get this version?
Are you still developing the software?

I would be very happy about an answer. Thanks a lot!

Best regards!

**Comment by take4blue** [comment]
to @Nastras 
I made an installer and published it on the following page.

https://take4-blue.com/en/adventurer3-controller/#toc36

Until V1.4, the source was not published on github, so I published the installer on my site.
Currently, I have released the source on github, so I stopped creating the installer.

I have a stance of making tools if I have what I need, and since there are no problems so far, I have stopped development.

In the future, if there is a problem with 3D printing due to the version upgrade of FlashPrint, Simplify3D or the firmware upgrade of Adventurer3, the tool may be updated.
